other/experiments/create_interface_with_min_energy_by_miller_indices.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>Same as above - the header should be h2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>